src/uncrustify_types.h

```diff
@@ -156,6 +156,7 @@ struct indent_ptr_t
 #define PCF_WF_ENDIF           PCF_BIT(39)  //! #endif for whole file ifdef
 #define PCF_IN_QT_MACRO        PCF_BIT(40)  //! in a QT-macro, i.e. SIGNAL, SLOT
 #define PCF_IN_FCN_CTOR        PCF_BIT(41)  //! inside function constructor
+#define PCF_IN_TRY_BLOCK       PCF_BIT(42)  //! inside Function-try-block
```

Comment:
Ack... aside from the conflicts with #2424, I also have a pending branch that needs a new flag :wink:.